docs/pages/Configuration.md

```diff
@@ -702,6 +703,19 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.0 [beta]
+
+!!! info
+    **Note:** UI5 Tooling version 3.0 is currently in development. If you wish to migrate to the latest UI5 Tooling, please check the [Upgrade Guide for v3](https://sap.github.io/ui5-tooling/v3/updates/migrate-v3/)
+
+**Breaking changes:**
+
+- Removes the `/proxy` endpoint and the corresponding
```

Comment:
this is a general breaking change, not related to the specVersion. All projects, regardless of their specVersion won't be able to use this feature anymore starting with v3.0

Reply:
Removed